presets/mainnet/network.yml

```diff
@@ -31,7 +31,7 @@ minVotingKeyLifetime: 112
 maxVotingKeyLifetime: 360
 votingKeyDesiredLifetime: 360
 votingKeyDesiredFutureLifetime: 60
-lastKnownNetworkEpoch: 467
+lastKnownNetworkEpoch: 488
```

Comment:
these configuration changes seem unrelated to adding a logger and shouldn't really be mixed into logging changes imo

Reply:
removed, those numbers are automatically updated when running the tests.